Added Bitbucket to Social Widgets

troubleshooter opened the pull request:
Hello,

I added Bitbucket to the list of social widgets to use/choose by updating the social.html widget. I also added the bitbucket variable to the README.md.

Thanks for your work.

Terry

Vimux commented:
Hello,

Maybe I am wrong, but It seems that you are using the Font Awesome icon (SIL OFL 1.1). Please, see #70.

troubleshooter commented:
Sorry. For some reason, I wasn't notified of your comment. I have no idea if it's a Font Awesome icon. A colleague gave me the svg code since I've no clue about svg files. I will enquire and also look at #70. 

Thanks.

troubleshooter commented:
I've updated the bitbucket icon using the information from the https://github.com/gilbarbara/logos file, corrected the fill color and run it through SVGO. 

Thanks for the pointers. 

Terry

Vimux commented on layouts/partials/widgets/social.html:
```diff
@@ -49,6 +49,14 @@ <h4 class="widget-social__title widget__title">{{ T "social_title" }}</h4>
 			</a>
 		</div>
 		{{- end }}
+		{{- with .Site.Params.widgets.social.bitbucket }}
```
1. Delete extra path in middle of icon (`M101.272`...)
1. Fix common fill attribute in svg tag (`fill="fff"` -> `fill="#fff"`)
1. Delete `fill`attributes in paths

troubleshooter replied:
I've made the changes. Thanks for your guidance.

Vimux replied:
Extra path (`M244.61`...) still exist

troubleshooter commented:
I've removed the second extra path. Thanks for your help.
